src/app/main.cpp

```diff
@@ -124,11 +130,46 @@ int main(int argc, char *argv[])
     setupDpi();
 #endif
 
+    // we need a copy of the arguments to extend them with the params file contents (see right below)
+    using args_vector = std::vector<char*>;
+    auto argsDeleter = [](args_vector *v)
```

Comment:
https://github.com/qbittorrent/qBittorrent/pull/9349#discussion_r213503918

Reply:
>#9349 (comment)

OK. Here is it:
>Shouldn't you free the vector elements when it is about to go out of scope?

It just a question. You could have just said "it is pointless" instead of doing this "pointless" thing.

Reply:
I decided then that without freeing the code would produce annoying false positives in memory analysers and went ahead adding the deleter.